bigtable/export_test.go

```diff
@@ -54,6 +54,7 @@ func init() {
 	flag.StringVar(&c.AdminEndpoint, "it.admin-endpoint", "", "Admin api host and port")
 	flag.StringVar(&c.DataEndpoint, "it.data-endpoint", "", "Data api host and port")
 	flag.StringVar(&c.Project, "it.project", "", "Project to use for integration test")
+	flag.StringVar(&c.Project2, "it.project2", "", "Second project to use for integration test")
```

Comment:
again would be nice to explain this is only for copy backup testing

Reply:
Done